metricflow/specs/where_filter_time_dimension.py

```diff
@@ -86,4 +94,6 @@ def _convert_to_time_dimension_spec(
             element_name=parameter_set.time_dimension_reference.element_name,
             entity_links=parameter_set.entity_path,
             time_granularity=parameter_set.time_granularity,
+            # TODO: add date_part to TimeDimensionCallParameterSet in DSI
+            # date_part=parameter_set.date_part,
```

Comment:
I don't think we want to do this. That makes date_part a piece of the metric spec, so if we want this as a query-only construct it's probably best to keep it in MetricFlow.

Reply:
Sounds gooood!